src/code/code_800E4FE0.c

```diff
@@ -536,7 +518,7 @@ s32 func_800E5EDC(void) {
 void func_800E5F34(void) {
     // macro?
     // clang-format off
-    s32 chk = -1; s32 sp28; do { } while (osRecvMesg(gAudioContext.audioResetQueueP, &sp28, 0) != chk);
+    s32 chk = -1;s32 sp28;do { } while (osRecvMesg(gAudioContext.audioResetQueueP, (OSMesg*) &sp28, OS_MESG_NOBLOCK) != chk);
```

Comment:
I think it should be formatted like this? I imagine it doesn't matter for codegen and it would match our normal formatting a bit more
```suggestion
    s32 chk = -1; s32 sp28; do { } while (osRecvMesg(gAudioContext.audioResetQueueP, (OSMesg*)&sp28, OS_MESG_NOBLOCK) != chk);
```